ci_pod/README.md

```diff
@@ -0,0 +1,21 @@
+Instructions for ci_pod deployment:
+
+1. Run `dnf install -y podman`
+2. Fill all the secrets in create_ci_pod.sh
+3. replace MANUALLY the following URLs in nginx/ibm_conf.conf
+     ```sh
+     JUPYTERLAB_URL=""
+     RUN_ARTIFACTS_URL=""
+     GRAFANA_URL=""
+     KIBANA_URL=""
+     ELASTICSEARCH_URL=""
+     INTERNAL_URL=""
+     ```
+4. Add ibm certificate files under ci_pod/nginx/:
+     ```sh
+     ibm_crt.crt;
+     ibm_private.key;
+    ```
+5. Create separate public ip for ci pod and update POD_CI_IP in create_ci_pod.sh
+6. Run `cp -rp ci_pod "$CI_POD_PATH"`
+7. Run `$CI_POD_PATH/ci_pod/./create_ci_pod.sh`
```

Comment:
No need for the "run" here; if you simply provide the command, that's fine, and that's conventionally how to present it.  So...

6. `cp -rp ci_pod "$CI_POD_PATH"`
7. `"$CI_POD_PATH/ci_pod/create_ci_pod.sh"`

(there's no need for the `/./`, which is a no op, but you do need to quote $CI_POD_PATH wherever you reference it).

Reply:
Fixed